sources/web/datalab/polymer/components/directory-picker-dialog/directory-picker-dialog.ts

```diff
@@ -34,8 +35,32 @@ class DirectoryPickerDialogElement extends BaseDialogElement {
 
   private static _memoizedTemplate: PolymerTemplate;
 
+  /**
+   * Initial value of input box.
+   */
+  public inputValue: string;
```

Comment:
I think it would be better to call this something like fileNameValue, and likewise call the boolean withFileName and update the names of the html fields to be more specific.